CONTRIBUTING.md

```diff
@@ -1,4 +1,5 @@
-# Contribution Guide #
+# Contribution Guide
```

Comment:
I am not sure what GitHub markdown syntax is. But I am pretty sure `#...#` is valid markdown syntax. Any particular reason you want to use this syntax? 
What is the difference between pre-change and post-change?